README.md

```diff
@@ -109,27 +143,27 @@ python -m tf2onnx.convert
     [--verbose]
 ```
 
-## Parameters
-### --input or --graphdef
+### Parameters
+#### --saved-model
+TensorFlow model as saved_model. We expect the path to the saved_model directory. tf2onnx will try to freeze the graph.
+#### --input or --graphdef
 TensorFlow model as graphdef file. If not already frozen we'll try to freeze the model.
 More information about freezing can be found here: [freeze graph tool](#freeze_graph).
```

Comment:
Thanks for looking into this -- re-reading it, it seems like some clarifications are needed!

Is it possible to convey to the user that they should only use only 1 of the following 4 parameters. Each one specifies the location of the input model file, but also tells tf2onnx what format to use to parse the model file  -- so if they specify multiple of these, it's just weird (@guschmue , should we change code to thrown an exception? Not sure if there's some history behind this however).

 Also, the ordering below is from "most common" to "least common", so it'd be good to have the ordering preserved.

   * --input 
   * --graphdef  (same description as --input, but used for forward compatibility)
   * --checkpoint
   * --saved-model

Reply:
We have a check to make sure there is at least 1 format but maybe we should check for exactly 1
https://github.com/onnx/tensorflow-onnx/blob/master/tf2onnx/convert.py#L79